sdk/src/main/java/com/horizen/box/data/CrossChainRedeemMessageBoxData.java

```diff
@@ -0,0 +1,67 @@
+package com.horizen.box.data;
+
+import com.horizen.box.CrossChainRedeemMessageBox;
+import com.horizen.proposition.PublicKey25519Proposition;
+import com.horizen.sc2sc.CrossChainMessage;
+
+public class CrossChainRedeemMessageBoxData
+        extends AbstractBoxData<PublicKey25519Proposition, CrossChainRedeemMessageBox, CrossChainRedeemMessageBoxData> {
+    private final CrossChainMessage message;
+    private final byte[] certificateDataHash;
+    private final byte[] nextCertificateDataHash;
+    private final byte[] scCommitmentTreeRoot;
+    private final byte[] nextScCommitmentTreeRoot;
+    private final byte[] proof;
+
+    public  CrossChainRedeemMessageBoxData(
+            PublicKey25519Proposition proposition,
+            CrossChainMessage message,
+            byte[] certificateDataHash,
```

Comment:
can be postponed after we complete a successfull test with the cryptolib: probably the two certificate hashes can be removed, because the final cryptolib implementation requires just the two commitmentroots for proof verification

Reply:
This comment too, do we want to wait for the cryptolib to be officially available?

Reply:
same, postpone